frontend/src/scenes/notebooks/Notebook/notebookLogic.ts

```diff
@@ -161,8 +173,11 @@ export const notebookLogic = kea<notebookLogicType>([
             },
         ],
     })),
-    selectors({
-        shortId: [() => [(_, props) => props], (props): string => props.shortId],
+    selectors(({ props }) => ({
+        shortId: [
+            () => [(_, props) => props],
+            (props): string => (props.cachedNotebook ? props.cachedNotebook.short_id : props.shortId),
```

Comment:
Is there a reason that the `cachedNotebook.short_id` would not match that in the props?

Reply:
Only that there's nothing stopping a human making the mistake... I was tempted to get fancy with the type, but actually we should make the impossible unrepresentable